src/sentry/tasks/reports.py

```diff
@@ -636,41 +714,28 @@ def deliver_organization_user_report(timestamp, duration, organization_id, user_
         message.send()
 
 
-Point = namedtuple("Point", "resolved unresolved")
-DistributionType = namedtuple("DistributionType", "label color")
-
-
 def series_map(function, series):
     return [(timestamp, function(value)) for timestamp, value in series]
 
 
-project_breakdown_colors = ["#422C6E", "#895289", "#D6567F", "#F38150", "#F2B713"]
+class Key(NamedTuple):
+    label: str
+    url: str
+    color: str
+    data: Mapping[str, int]
 
-total_color = """
-linear-gradient(
-    -45deg,
-    #ccc 25%,
-    transparent 25%,
-    transparent 50%,
-    #ccc 50%,
-    #ccc 75%,
-    transparent 75%,
-    transparent
-);
-"""
 
+class Point(NamedTuple):
+    resolved: int
+    unresolved: int
 
-def build_project_breakdown_series(reports):
-    Key = namedtuple("Key", "label url color data")
 
-    def get_legend_data(report):
```

Comment:
Just killed this since we don't compute events from the series anymore, we just use the accepted count